src/enums.h

```diff
@@ -245,5 +245,15 @@ typedef enum {
 } htcp_clr_reason;
 #endif /* USE_HTCP */
 
+typedef enum {
```

Comment:
We are trying to get rid of `enums.h` so it is probably best to place this declaration elsewhere. We could add a dedicated header (e.g., `src/XactionSteps.h`) or perhaps (ab)use `src/MasterXaction.h` a little.

Reply:
commit ccd68c4b68